source/NVDAObjects/UIA/edge.py

```diff
@@ -458,7 +458,7 @@ def _get_ariaProperties(self):
 	# in a list of strings like "something=true;current=date;". We want to capture one group, after the '='
 	# character and before the ';' character.
 	# This could be one of: True, "page", "step", "location", "date", "time"

```

Comment:
Could you update this comment to add "false" as a valid value?

Reply:
I also double checked in edge. `True` should be listed as "true"